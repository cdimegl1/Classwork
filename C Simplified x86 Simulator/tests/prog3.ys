#Fig 4.45, pg. 431 of text
prog2:
	irmovq	$10, %rdx
	irmovq  $3, %rax
	nop
	addq	%rdx, %rax
	halt

