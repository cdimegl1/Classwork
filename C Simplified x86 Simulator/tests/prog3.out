   0:	0000	bubble
   1:	0000	bubble
   2:	0000	bubble
   3:	0000	bubble
   4:	0000	irmovq	$0xa, %rdx
   5:	000a	irmovq	$0x3, %rax
   6:	0014	nop	
   7:	0015	bubble
   8:	0015	bubble
   9:	0015	addq	%rdx, %rax
  10:	0017	halt	
rax: 000000000000000d
rcx: 0000000000000000
rdx: 000000000000000a
rbx: 0000000000000000
rsp: 0000000000000000
rbp: 0000000000000000
rsi: 0000000000000000
rdi: 0000000000000000
 r8: 0000000000000000
 r9: 0000000000000000
r10: 0000000000000000
r11: 0000000000000000
r12: 0000000000000000
r13: 0000000000000000
r14: 0000000000000000
 pc: 0000000000000017
status: HLT
cc: Z=0 S=0 O=0
